ovssubnet/common.go

```diff
@@ -312,7 +314,12 @@ func (oc *OvsController) StartNode(sync, skipsetup bool) error {
 			log.Errorf("Failed to obtain ContainerNetwork: %v", err)
 			return err
 		}
-		err = oc.flowController.Setup(oc.localSubnet.Sub, containerNetwork)
+		servicesNetwork, err := oc.subnetRegistry.GetServicesNetwork()
```

Comment:
flowController.Setup() has a servicesNetwork arg now though, so we have to pass something, even in the non-multitenant case. And GetServicesNetwork() will return the right answer whether we're using multitenant or not. So even though kube and lbr don't use it, it doesn't hurt (and they could theoretically use it in the future).